### PR TITLE
Spooky Scary Skeletons!

### PR DESCRIPTION
**Pull request by Memendia**
Not tested 100% sure it works

### CODE REVIEW

File: Tcomms EXPERIMENTAL.txt [hunk 1]
```diff
@@ -1236,4 +1264,4 @@ if($eS != "-" && $IsAIorBorg && $NOSPAM != 1 && ($OrigFreq == $Tfq || $OrigFreq
 
 elseif($IsAIorBorg && ($eP1 == "-" || $eP2 == "-"|| $NOSPAM != 2) && $OrigFreq != 1357) { $freq=0;}
 //Perhaps the most Frankensteined Tcomms script you will ever see.
-//Credits: Magni Bronzebeard (Author), Giacomand, SirBayer, MisterPerson, wiki.ss13.eu
\ No newline at end of file
+//Credits: Magni Bronzebeard (Author), Giacomand, SirBayer, MisterPerson, wiki.ss13.eu
```
Comment (GunHog) [inline Tcomms EXPERIMENTAL.txt]: Remove MisterPerson while you are at it. I keep forgetting to purge that jerk.

Comment (Memendia) [inline Tcomms EXPERIMENTAL.txt]: Can I add myself in after I remove him? :D

Comment (GunHog) [inline Tcomms EXPERIMENTAL.txt]: I guess.

File: Tcomms EXPERIMENTAL.txt [hunk 2]
```diff
@@ -678,6 +697,10 @@ if($Ex1=="-mutemode") {
 	toggle("SpecMute","Standard Mute Active.","Special Mute Active.");
 }
 
+/*
```
Comment (GunHog) [inline Tcomms EXPERIMENTAL.txt]: Do not add comments like this, there is a space limit for NTSL, and it is wasted on anything that is not code or critical comments.

File: Tcomms EXPERIMENTAL.txt (hunk 1 above)
Comment (Memendia) [inline Tcomms EXPERIMENTAL.txt]: Wooo

File: Tcomms EXPERIMENTAL.txt (hunk 2 above)
Comment (Memendia) [inline Tcomms EXPERIMENTAL.txt]: Alrightio

**Comment by Memendia** [comment]
@GunHog All ready to go boss :+1:

**Comment by GunHog** [comment]
Did you test it? All of it? I want screenshots.

**Comment by Memendia** [comment]
:-1: will test

**Comment by Memendia** [comment]
:-1: doesn't work apparently and not enough room in the script editor

**Comment by GunHog** [comment]
This is why I tell you to test things, and why I told you not to waste space :)

**Comment by Memendia** [comment]
Not borken!

**Comment by Memendia** [comment]
@GunHog Skeleton was tested and works

**Comment by GunHog** [comment]
Bueno. Do not disappoint me, Nex!

**Comment by Memendia** [comment]
D: I forgot the proof of it working
honk

**Pull request merged**